File: examples/ex012_preço_com_5_por_cento_desconto/ex012.js
function calculate() {

    let inputPrice = document.querySelector('#input1').value;
    let inputPriceNumber = Number(inputPrice);
    let divAnswer = document.querySelector('.answer');

    if (inputPrice == "" || inputPrice <= -1) {

        divAnswer.innerHTML = `<div class="answer error">INFORME UM VALOR VÁLIDO!</div>`;

    } else {

        divAnswer.innerHTML = "";
        divAnswer.innerHTML += `<div class="answer">O preço atual é <strong>R$ ${inputPriceNumber.toFixed(2)} reias</strong>!</div>`;
        divAnswer.innerHTML += `<div class="answer">O preço do seu produto com 5% de desconto fica </br> <strong>R$ ${(inputPriceNumber - (inputPriceNumber*5/100)).toFixed(2)} reais</strong>! </br> Boas Compras!!!</div>`;

    }
}

File: examples/2_tratando_dados_fazendo_contas/ex012_preço_com_5_por_cento_desconto/ex012.py
# faça um algoritmo que leia o preço de um produto e mostre seu novo preço  com 5% de desconto.

price = float(input('Digite o preço atual: R$'))

print(f'O preço atual do produto é R$ {price:.2f} reais!')
print(f'O valor da mercadoria com 5% de desconto é R$ {(price - (price*5/100)):.2f} reais!')
print(f'Boa compras!')
